src/components/scenes/FioAddressDetailsScene.js

```diff
@@ -60,23 +61,40 @@ class FioAddressDetails extends React.Component<Props, LocalState> {
 
   _onPressAccountSettings = (): void => {
     const { navigation, route } = this.props
-    const { fioAddressName } = route.params
+    const { fioAddressName, bundles } = route.params
     const { fioWallet } = this.state
     if (fioWallet) {
       navigation.navigate('fioAddressSettings', {
         fioWallet,
-        fioAddressName
-        // todo: refreshAfterBundlesAdded
+        fioAddressName,
+        bundles,
```

Comment:
Shouldn't `showAddBundles` be set to `true` here?

Reply:
FioAddressSettings scene has two actions user could choose - Transfer or Add Bundles. `showAddBundles` prop is used to skip selection and show Add Bundles action when going to the scene